packages/calcite-components/src/components/text-area/text-area.scss

```diff
@@ -1,66 +1,87 @@
+@import "~@esri/calcite-design-tokens/dist/scss/core";
+
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-text-area-background-color: Specifies the background color of the component.
+ * @prop --calcite--text-area-border-block-end-color: Specifies the border block end color of the component.
```

Comment:
change to `--calcite-text-area-border-color`

Reply:
we have `--calcite-text-area-border-color` specified already. The `border-block-end-color` is different from the `border-color` for a valid text-area we need to define them separate.